static/src/javascripts/projects/common/modules/commercial/create-ad-slot.js

```diff
@@ -94,25 +94,33 @@ define([
 
     return function (name, types, series, keywords, slotTarget) {
         var attrName,
-            definition = adSlotDefinitions[slotTarget ? slotTarget : name],
-            dataAttrs  = {
-                label:   definition.label !== undefined ? definition.label : true,
-                refresh: definition.refresh !== undefined ? definition.refresh : true
-            },
-            $adSlot = $.create(template(
-                adSlotTpl,
-                {
-                    name: definition.name || name,
-                    // badges now append their index to the name
-                    normalisedName: (definition.name || name).replace(/((?:ad|fo|sp)badge).*/, '$1'),
-                    types: types ? map((isArray(types) ? types : [types]), function (type) {
-                        return ' ad-slot--' + type;
-                    }).join('') : '',
-                    sizeMappings: map(pairs(definition.sizeMappings), function (size) {
-                        return ' data-' + size[0] + '="' + size[1] + '"';
-                    }).join('')
-                })
-            );
+            slotName = slotTarget ? slotTarget : name,
+            definition,
+            dataAttrs,
+            $adSlot;
+
+        definition = adSlotDefinitions[slotName];
+        if (config.page.hasPageSkin && slotName === 'merchandising-high') {
+            definition.sizeMappings.wide = '1,1';
+        }
+        dataAttrs  = {
+            label:   definition.label !== undefined ? definition.label : true,
+            refresh: definition.refresh !== undefined ? definition.refresh : true
```

Comment:
`definition.label || true` is simpler, but it depends whether you want other falsy values. JS IS WEIRD.